packages/stark-ui/src/modules/input-mask-directives/directives/text-mask.constants.ts

```diff
@@ -1,5 +1,6 @@
 /**
  * Util class containing some constants with predefined masks for common mask patterns to be used with the {@link StarkTextMaskDirective}.
+ * @dynamic
```

Comment:
Why is this added here? Can you add a comment or a link where we should look at to understand this?